### PR TITLE
add 2020 recap post

### PR DESCRIPTION
**Pull request by narimiran**

### CODE REVIEW

File: jekyll/_posts/2020-12-28-nim-in-2020-a-short-recap.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+Due to COVID-19 situation, we have decided that the first ever Nim conference
+should be held online and be available for free for everybody.
+
+It was held on June 20th via Youtube streams, containing **15 interesting talks**
+where the authors showed wide variety of things they are developing using Nim.
+Every talk also had a live chat where the authors and the audience interacted,
+which proved to be very useful to get answers to some specific questions and
+go into more depth about the topic.
+
+All talks can still be viewed on
```
Comment (ee7) [inline jekyll/_posts/2020-12-28-nim-in-2020-a-short-recap.md:41]:
```suggestion
All the talks can still be viewed on
```

File: jekyll/_posts/2020-12-28-nim-in-2020-a-short-recap.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+[the Youtube playlist](https://www.youtube.com/playlist?list=PLxLdEZg8DRwTIEzUpfaIcBqhsj09mLWHx)
+and we recommend you to take a look if you haven't watched the talks already.
```
Comment (ee7) [inline jekyll/_posts/2020-12-28-nim-in-2020-a-short-recap.md:43]:
```suggestion
and we recommend that you take a look if you haven't watched them already.
```

**Comment by narimiran** [comment]
@ee7 thank you very much for your detailed review! As you can tell - I'm not a native speaker :)

**Review by dom96 — APPROVED** [review]
Nice!

**Pull request merged**